README.md

```diff
@@ -36,6 +36,17 @@ Includes [Jenkins Pipeline](https://github.com/jenkinsci/workflow-plugin) suppor
 slackSend color: 'good', message: 'Message from Jenkins Pipeline'
 ```
 
+Additionally you can pass a JSONArray as a String in order to send complex messages
```

Comment:
Please include a link to https://api.slack.com/docs/message-attachments since this is specifically referring to that feature.  It would allow users to more easily learn about it.